wagtail/tests/testapp/fixtures/test.json

```diff
@@ -87,6 +87,41 @@
 },
 
 {
+    "pk": 13,
```

Comment:
Changing line 65 of this file to ``"numchild": 1`` might solve the issue with the fixtree command detecting orphans